ansys/tools/example_coverage/example_coverage.py

```diff
@@ -0,0 +1,156 @@
+import ast
+import os
+import sys
+import argparse
+import pathlib
+
+def find_files(folder_path):
+    """Find all modules available in the folder path provided.
+    The selection is made based on the file's extension.
+    However, __init__.py will be discarded."""
+
+    # Raise an exception if the folder is empty.
+    if len(os.listdir(folder_path)) == 0:
+        raise Exception(f"None file nor folder available in {folder_paths}.")
+
+    module_extension = ('.py')
+    modules = []
+
+    for path, subdirs, files in os.walk(folder_path):
+        for name in files:
+            if name.endswith(module_extension) & (not name.endswith("__init__.py")):
```

Comment:
```suggestion
            if name.endswith('.py'):
```
We should include `__init__.py` anyway.  People might decide to add code here.